task04/test/unit/helpers.test.ts

```diff
@@ -0,0 +1,35 @@
+import {validateInput, shortenPublicHoliday} from '../../helpers'
+import {holidays, shortHolidays} from '../test-data'
+import {SUPPORTED_COUNTRIES} from "../../config";
+
+const year = 2024;
+const country = SUPPORTED_COUNTRIES[0];
+
+describe('Validate input', () => {
+  test('should return true if year and country are correct',  () => {
+    const inputValidationResult = validateInput({year, country});
+    expect(inputValidationResult).toBe(true);
+  });
+
+  test('should return true if country is correct',  () => {
+    const inputValidationResult = validateInput({country});
+    expect(inputValidationResult).toBe(true);
+  });
+
+  test('should throw error if country is invalid', () => {
+    const invalidCountry = "GB1"
+    expect(() => validateInput({country: invalidCountry})).toThrow(`Country provided is not supported, received: ${invalidCountry}`);
```

Comment:
FYI: In practice it's better not to be tied up in the message, it's better to check the type of error being produced, but you'll be working with custom errors later, so the comment is just for information.